packages/core/src/errors.ts

```diff
@@ -25,3 +25,11 @@ export class NotFoundError extends Error {
     super(message)
   }
 }
+
+export class PaginationPayloadInvalidError extends Error {
+  name = "PaginationPayloadInvalidError"
+  statusCode = 422
+  constructor(message = "The skip or take parameters are invalid") {
```

Comment:
```suggestion
  constructor(message = "The pagination arguments are invalid") {
```

Remove the specific fields here so that the base is generic, and then you can add a specific message when you throw it.